README.md

```diff
@@ -14,8 +14,15 @@ user! All RSS items will be posted as toots.
 - `@engadget.com@mastofeeder.com`
 - `@techcrunch.com@mastofeeder.com`
 - Or any other website that has an RSS feed! `@YOUR_WEBSITE_HERE@mastofeeder.com`
-- Convert slashes to dots: `indiegames.com/blog` -> `@indiegames.com.blog@mastofeeder.com`
-- Remove `.rss` and `.xml` extensions: `https://feeds.yle.fi/uutiset/v1/majorHeadlines/YLE_UUTISET.rss` -> `@feeds.yle.fi.uutiset.v1.majorHeadlines.YLE_UUTISET`
+
+You can follow feeds with slashes in the URL by replacing the slashes with `..`:
+
+- `indiegames.com/blog` -> `@indiegames.com..blog@mastofeeder.com`
+
+Some websites might have feed URLs that are case-sensitive or contain characters that can't be expressed in a Fediverse username. To follow these feeds, you can replace the first slash with `._.` and [base32 encode](https://www.rfctools.com/base32-encoder/) the path, ignoring any trailing `=`. For example:
+
+- `en.wikipedia.org/w/api.php?action=featuredfeed&feed=featured&feedformat=rss` -> ``
+`en.wikipedia.org._.O4XWC4DJFZYGQ4B7MFRXI2LPNY6WMZLBOR2XEZLEMZSWKZBGMZSWKZB5MZSWC5DVOJSWIJTGMVSWIZTPOJWWC5B5OJZXG@mastofeeder.com`
```

Comment:
Oh, this is a very interesting approach, I like it! I'm thinking that perhaps the main mastofeeder.com website could have an URL -> username converter for convenience 🤔

Reply:
This would definitely be nicer with a convenient converter! How were you envisioning it — an extra route with a GUI for the converter?

Reply:
Quick question, has the mastofeeder.com main website been discontinued - it redirects to the github repo page right now. 

I've got a nitter profile RSS feed I'd like to parse, but I can't although I tried the URL conversion you mentioned. This is probably because the url has no `.rss` or `.xml` extension at the end so this PR could help. https://nitter.poast.org/jo_chemla/with_replies/rss `@nitter.poast.org.jo_chemla.with_replies.rss@mastofeeder.com`

Reply:
Yeah, website root has always redirected to Github as an MVP website 😄 Sorry that this has been stale for so long, I have very little time to work on Mastofeeder right now

Reply:
No worries, thanks for that tool anyway! You confirm that nitter url format cannot be translated to something which mastofeeder expects, because there is no suffix to the rss feed url? Thanks!